metrics/web/test/common/presentation/navigation/metrics_page/metrics_page_factory_test.dart

```diff
@@ -16,153 +18,193 @@ import '../../../../test_utils/route_name_mock.dart';
 
 void main() {
   group("MetricsPageFactory", () {
-    const path = 'path';
+    const unknownRouteName = 'unknown';
+    const pageParameters = DashboardPageParametersModel();
+    const dashboardRouteName = RouteName.dashboard;
+    const projectGroupsRouteName = RouteName.projectGroups;
 
     final metricsPageFactory = MetricsPageFactory();
-    final routeConfiguration = RouteConfigurationMock();
     final routeName = RouteNameMock();
-
-    tearDown(() {
-      reset(routeConfiguration);
-      reset(routeName);
-    });
+    final routeConfiguration = RouteConfigurationMock();
```

Comment:
✅ 
It seems like we should reset mocks after every test case to avoid side effects and run test cases independently.